src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const sourcesArr = sourceString.split(';');
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. Consider renaming `sourcesArr` to a more descriptive name that reflects the content of the array, such as `styleDeclarations`.